androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -172,6 +179,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         new TwaSharedPreferencesManager(this)
                 .writeLastLaunchedProviderPackageName(mTwaLauncher.getProviderPackage());
+
+        mPackageManager = getPackageManager();
+        // Check whether ManageDataLauncherActivity is present in manifest.
+        Intent siteSettingsIntent = new Intent(this, ManageDataLauncherActivity.class);
+        siteSettingsIntent.setAction(ManageDataLauncherActivity.ACTION_MANAGE_TRUSTED_WEB_ACTIVITY_DATA);
```

Comment:
Do we need the action if we're creating the Intent with the class name?

Reply:
Yes, otherwise we receive a nullpointer with the message: intent's action must be set